src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -284,7 +284,7 @@ void trace_hostpolicy_entrypoint_invocation(const pal::string_t& entryPointName)
 // If hostpolicy is already initalized, the library will not be
 // reinitialized.
 //
-SHARED_API int corehost_load(host_interface_t* init)
+SHARED_API int __cdecl corehost_load(host_interface_t* init)
```

Comment:
Should we give hostpolicy the same treatment as hostfxr? Not necessarily with the header (although it would not hurt either), but with the define of calling convention... it feels weirdly inconsistent to have a define in one case and hard coded __cdecl in the other.

Reply:
Yeah, I'd say we should have a header so that at least hostpolicy, hostfxr, and mockhostpolicy could share it (not necessarily written to be usable externally, but at least one shared place internally).